lib/monitor/monitor.go

```diff
@@ -18,31 +18,51 @@ import (
 var Sink = "0.0.0.0"
```

Comment:
One could also go with `Measure` + `Measures` but the difference might be too subtle? Maybe one of the following?
`Measure` + `MeasureCollection`
`Measurement` + `MeasurementCollection`
`Record` + `RecordCollection`
`DataPoint` + `DataPointCollection`
Too clumsy?